ssz/__init__.py

```diff
@@ -1,113 +1,42 @@
+#
+# sedes
+#
 from .codec import (  # noqa: F401
+    decode,
     encode,
-    decode
 )
-
 from .exceptions import (  # noqa: F401
-    SSZException,
-    SerializationError,
     DeserializationError,
+    SerializationError,
+    SSZException,
 )
-
-from .tree_hash import (  # noqa: F401
-    hash_tree_root,
-)
-
-#
-# sedes
-#
 from .sedes import (  # noqa: F401
     BaseSedes,
-    FixedSizedSedes,
-    LengthPrefixedSedes,
-
+    BasicSedes,
+    Boolean,
+    Byte,
+    ByteList,
+    ByteVector,
+    CompositeSedes,
     Container,
-    Serializable,
-
     List,
-    empty_list,
-
-    Boolean,
+    Serializable,
+    UInt,
+    Vector,
     boolean,
-
-    Bytes,
-    BytesN,
-    bytes_sedes,
+    byte_list,
+    byte_vector,
     bytes32,
```

Comment:
Can I get a `bytes4` since it's used in the spec?

Reply:
Added `bytes64`, reread the comment, removed `bytes64`, added `bytes4`.